user/openmandriva/macros

```diff
@@ -154,7 +154,7 @@ Provides:	%{1} = %{?2}%{!?2:%{EVRD}} \
 # cf http://wiki.mandriva.com/en/Development/Packaging/Problems#format_not_a_string_literal_and_no_format_arguments
 %Werror_cflags -Wformat -Werror=format-security
 
-%_ssp_cflags -fstack-protector-strong --param=ssp-buffer-size=4 %{?_serverbuild_flags: %_serverbuild_flags}
+%_ssp_cflags -fstack-protector-strong --param=ssp-buffer-size=4%{?_serverbuild_flags: %_serverbuild_flags}
```

Comment:
Could you change it to remove the space inside the conditional rather than outside the conditional?

Reply:
Should be just {?_serverbuild_flags}, I guess. I changed the line to RPM5's value in my original commit.